edge-apps/grafana/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://json.schemastore.org/prettierrc",
+  "semi": false,
+  "singleQuote": true,
+  "printWidth": 80
```

Comment:
Can we move it to the top level and create a workflow to make sure that all Edge Apps are consistent?